web_services/foxx/brightid/manifest_apply.json

```diff
@@ -2,7 +2,7 @@
   "main": "apply.js",
   "name": "apply",
   "description": "Allows BrightID consensus module to apply operations to the database.",
-  "version": "4.6.2",
+  "version": "5.4.0",
```

Comment:
why change the version of brightid4/manifest_apply.json to `5.4.0.`?

Reply:
No, github compare confused and wrongly compared `brightid/mainfest_apply.json` to `brightid4` instead of `brightid5`